project/src/components/Splash/Splash.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import "./Splash.css";
+import { useState } from 'react';
+
+const Splash = () => {
+  const [showSplash, setShowSplash] = useState(true);
+  (() => {
+  setTimeout(() => {
+    setShowSplash(false)}
```

Comment:
this is a cool auto-scroll! and, use caution about running this scroll no matter what after 7 seconds - if someone is reading farther down the page and gets scrolled back up mid-way, this could be a tougher user experience